.github/workflows/test-turbo.yml

```diff
@@ -0,0 +1,189 @@
+name: Symfony UX Turbo
```

Comment:
Is it necessary to have a new CI file?

Reply:
It allows to not set up a useless Mercure service for all other bundles (https://github.com/symfony/ux/pull/64/files/e8b335bd5fb7d50b209fa2e832f22fd846c16bbe#diff-b65cafb402a3698791b8c52f1f17bd1887642e197a7d453c6c3379910f06bd3fR51), but I can merge both files if you prefer.